ui/design/src/components/Editor/index.tsx

```diff
@@ -212,7 +213,7 @@ const EditorBox: React.FC<IEditorBox> = React.forwardRef((props, ref) => {
    * set the selection at the end of the content when component is mounted
    */
   useEffect(() => {
-    Transforms.move(editor);
+    Transforms.move(editorRef.current);
```

Comment:
does this change anything? since editor is already assigned to editorRef.current it should be the same thing, no?

Reply:
it's not the same thing! the reference is different when comparing refs and ref.current!

Reply:
I was refering to this line https://github.com/AKASHAorg/akasha-world-framework/pull/1524/files#diff-630fe981e4c6d0089d0a1a77d977991138f23c824be3c6f3857f93b1f30c8cb6R183 above, so editor = editorRef.current

Reply:
editorRef.current can change between renders, you can modify it `editorRef.current = new Editor...`. That's why it's expected to use the actual ref in hook!

Reply:
ok, thanks for the explanation